Examples/VisionExamples/Examples/ARCustomizationViewController.swift

```diff
@@ -0,0 +1,146 @@
+            // Set the color of the fence
+            fenceVisualParams.color = UIColor.green
+            // Set the height of the fence in meters
+            fenceVisualParams.height = 3
+            // Set the vertical offset of the fence above the road surface in meters
+            fenceVisualParams.verticalOffset = 0
+            // Set the horizontal offset of the fence from the center of the road in meters
+            fenceVisualParams.horizontalOffset = 5
+            // Set the number of arrows in the fence
+            fenceVisualParams.sectionsCount = 4
+            // After `fenceVisualParams` is configured you need to set it to the view controller
+            visionARViewController.set(fenceVisualParams: fenceVisualParams)
+        },
+        // Let's customize fence with different visual params to see the difference
+        { visionARManager, visionARViewController in
+            let fenceVisualParams = FenceVisualParams()
+            fenceVisualParams.color = UIColor.red
+            fenceVisualParams.height = 2
+            fenceVisualParams.verticalOffset = 1.5
+            fenceVisualParams.horizontalOffset = 2
+            fenceVisualParams.sectionsCount = 5
+            visionARViewController.set(fenceVisualParams: fenceVisualParams)
+        }, { visionARManager, visionARViewController in
+            let fenceVisualParams = FenceVisualParams()
+            fenceVisualParams.color = UIColor.brown
+            fenceVisualParams.height = 1.5
+            fenceVisualParams.verticalOffset = 0.5
+            fenceVisualParams.horizontalOffset = 3
+            fenceVisualParams.sectionsCount = 6
+            // Let's show lane as well
+            visionARViewController.isLaneVisible = true
+            visionARViewController.set(fenceVisualParams: fenceVisualParams)
+        }, { visionARManager, visionARViewController in
+            let fenceVisualParams = FenceVisualParams()
+            fenceVisualParams.color = UIColor.yellow
+            fenceVisualParams.height = 4
+            fenceVisualParams.verticalOffset = 0
+            fenceVisualParams.horizontalOffset = 5
+            fenceVisualParams.sectionsCount = 3
+            visionARViewController.set(fenceVisualParams: fenceVisualParams)
+        }, { visionARManager, visionARViewController in
+            // Set fence visual params to defaults. Setting fence visual params doesn't visibility distance
+            visionARViewController.set(fenceVisualParams: FenceVisualParams())
+        }
+    ]
+
+    var customizationIndex: Int = 0
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        // In order to show all the possible customizations, which is not possible at one time, let's use timer
+        // to switch styles every second to see the difference between different settings.
+        let timer = Timer.scheduledTimer(withTimeInterval: 1.0, repeats: true) {[weak self] timer in
```

Comment:
Should we increase default time interval a bit? Say, 3 seconds?

Reply:
I tried 5 seconds, it's too boring to wait for style switching so much time